generative_ai/safety/safety_config_example.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+def generate_text() -> str:
+    # [START generativeaionvertexai_gemini_safety_settings]
+    import vertexai
+
+    from vertexai.generative_models import (
+        GenerativeModel,
+        HarmCategory,
+        HarmBlockThreshold,
+        Part,
+        SafetySetting,
+    )
+
+    # TODO(developer): Update and un-comment below line
+    # PROJECT_ID = "your-project-id"
+    vertexai.init(project=PROJECT_ID, location="us-central1")
+
+    model = GenerativeModel("gemini-1.5-flash-001")
+
+    # Safety config
+    safety_config = [
+        SafetySetting(
+            category=HarmCategory.HARM_CATEGORY_DANGEROUS_CONTENT,
+            threshold=HarmBlockThreshold.BLOCK_LOW_AND_ABOVE,
+        ),
+        SafetySetting(
+            category=HarmCategory.HARM_CATEGORY_HARASSMENT,
+            threshold=HarmBlockThreshold.BLOCK_LOW_AND_ABOVE,
+        ),
+    ]
+
+    image_file = Part.from_uri(
+        "gs://cloud-samples-data/generative-ai/image/scones.jpg", "image/jpeg"
+    )
+
+    # Generate content
+    responses = model.generate_content(
+        [image_file, "What is in this image?"],
+        safety_settings=safety_config,
+        stream=True,
+    )
+
+    text_responses = []
+    for response in responses:
+        print(response.text)
+        text_responses.append(response.text)
+    # Example response:
+    # The image contains a plate of blueberry scones with a cup of coffee and a spoon that
+    # says "let's jam". There are fresh blueberries in a bowl, and a bouquet of pink peonies.
+    # The image looks like a beautiful and delicious breakfast or brunch spread.
+
+    # [END generativeaionvertexai_gemini_safety_settings]
+    return "".join(text_responses)
```

Comment:
Nit: Is there a reason to use the `stream` parameter since we're just joining the responses at the end?

It also doesn't print out the safety detections.

Maybe change this to not use `stream` and just print out the full response object. Or print `response.text` and `response.candidates[0].safety_ratings`

```suggestion
    response = model.generate_content(
        [image_file, "What is in this image?"], safety_settings=safety_config
    )

    print(response.text)
    print(response.candidates[0].safety_ratings)
    # Example response:
    # The image contains a plate of blueberry scones with a cup of coffee and a spoon that
    # says "let's jam". There are fresh blueberries in a bowl, and a bouquet of pink peonies.
    # The image looks like a beautiful and delicious breakfast or brunch spread.

    # [END generativeaionvertexai_gemini_safety_settings]
    return response.text
```

Reply:
IMO, there are no reason to use `stream` parameter. 
I modified the code and provided a new example for response.